agent/seleniumlibrary-firefox/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+export DISPLAY=:0
+rm -rf  /tmp/.X0-lock  # To remove the lock in case where the GUI server is already running
+GEOMETRY="${SCREEN_WIDTH}x${SCREEN_HEIGHT}x${SCREEN_DEPTH}"
+Xvfb :0 -screen 0 $GEOMETRY -nolisten tcp &
+echo "Test is launched"
+robot -d /tmp/reports/ /tmp/selenium_firefox_test.robot || exit 1
```

Comment:
Running robot not rfswarm-agent? is this what you intended or was this something you were testing with?

Reply:
Yeah this was just a test, I wanted to check firefox success launch in headless mode. We're going to add agent test, since Ksenia recently finished manager setting in github runner. Then, I might be able to add some tests with the agent for all images versions.

Reply:
So do you plan on changing it back to running the agent before merging into master?

Reply:
Yes, of course. Ksenia made the same remark to me. I intend to remove the launch of entrypoint.sh file in the dockerfile, since the agent can already be launched from the agent base image and remove robot command line in the entrypoint.sh file as well. Thus, the entrypoint.sh could be use for test cases or to display IHM when needed.
Beside, I'm going to add some structure tests for all images decorrelated to the agent images generation.
What do you think?

Reply:
I think this entry point file over rides the entry point file from the base image, even if it doesn't replace the original file,  my understanding is that an image only has one entry point, so this file will be run and not the base image's file, so like this the image is no longer an agent image.

Happy to be shown I'm wrong, that's just my understanding of it.

Reply:
Quite right. So in your opinion I should add all the Xvfb configuration in the Dockerfile if I want to keep the display available in the container?

Reply:
Actually, no I think you should keep this entry point file, just replace the last 2 lines with the lines from the base image entry point. i.e.
Replace:
```
echo "Test is launched"
robot -d /tmp/reports/ /tmp/selenium_firefox_test.robot || exit 1
```

with:
```
DEFAULT_ARGS="-m http://manager:8138 -g 2 -p container:docker"
echo "Running rfswarm-agent..."
# Use the default args and overwrite them with user arguments
rfswarm-agent $DEFAULT_ARGS "$@"
```

So the entry point file for this image would become:
```
export DISPLAY=:0
rm -rf  /tmp/.X0-lock  # To remove the lock in case where the GUI server is already running
GEOMETRY="${SCREEN_WIDTH}x${SCREEN_HEIGHT}x${SCREEN_DEPTH}"
Xvfb :0 -screen 0 $GEOMETRY -nolisten tcp &
DEFAULT_ARGS="-m http://manager:8138 -g 2 -p container:docker"
echo "Running rfswarm-agent..."
# Use the default args and overwrite them with user arguments
rfswarm-agent $DEFAULT_ARGS "$@"
```

Reply:
I just removed test reference in entrypoint.sh file and add the agent launch. I'm going to create a tag and push it on dockerhub. Right away I'm going to look at the selenium-chrome-agent issue. Thanks you'll for your review.